src/openzaak/components/besluiten/admin.py

```diff
@@ -38,15 +38,25 @@ class BesluitInformatieObjectAdmin(
     AuditTrailAdminMixin, UUIDAdminMixin, admin.ModelAdmin
 ):
     form = BesluitInformatieObjectForm
-    list_display = ("besluit", "_informatieobject", "_informatieobject_url")
+    list_display = (
+        "besluit",
+        "_informatieobject",
+        "_informatieobject_base_url",
+        "_informatieobject_relative_url",
+    )
     list_filter = ("besluit",)
     search_fields = (
         "besluit__uuid",
         "_informatieobject__enkelvoudiginformatieobject__uuid",
-        "_informatieobject_url",
+        "_informatieobject_relative_url",
```

Comment:
can we do an annotation on the queryset that concacts the service root + relative URL, so we can still search on fully qualified URLs?

Reply:
annotations are added